extensions/amp-ad/0.1/test/test-concurrent-load.js

```diff
@@ -75,13 +75,27 @@ describes.realWin('concurrent-load', {}, env => {
       installTimerService(win);
     });
 
-    it('should throttle ad loading one per second', () => {
+    it('should throttle ad loading one per second', function* () {
       expect(is3pThrottled(win)).to.be.false;
       incrementLoadingAds(win);
       expect(is3pThrottled(win)).to.be.true;
       clock.tick(999);
+      yield;
```

Comment:
When code runs to a`yield` statement in a Generator, it stops and yield to other tasks/micro tasks in the event loop to run. It will resume once the Generator.next() is called. 

Calling `Generator.next()` in a setInterval makes sure the remaining work always get queued   to the end of event loop.